### PR TITLE
Add KUBECTL_KUBERC environment variable description

### PR DESCRIPTION
**Pull request by ardaguclu**
This PR adds new environment variable into the docs for https://github.com/kubernetes/enhancements/issues/3104

### CODE REVIEW

File: content/en/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -390,6 +390,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_KUBERC</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, kuberc file is taken account to define user specific preferences.
```
Comment (soltysh) [inline content/en/docs/reference/kubectl/kubectl.md:397]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, kuberc file is taken into account to define user specific preferences.  
```

Comment (ardaguclu) [reply content/en/docs/reference/kubectl/kubectl.md]: Updated. Thank you.

**Comment by stmcginnis** [comment]
Should wait on, but fast-follow, the merging of https://github.com/kubernetes/kubernetes/pull/125230

**Comment by ardaguclu** [comment]
We've missed the code expect and functionality is deferred to 1.33
/close

**Pull request closed without merging**